app/controllers/article_selections_controller.rb

```diff
@@ -0,0 +1,79 @@
+class ArticleSelectionsController < ApplicationController
```

Comment:
It would not take much convincing to get me to change this to ArticleBookmarks.  It's called "Selections" in the UI, so I started making things called that, but the Blacklight code refers to it as "Bookmarks".  If I do that, I'll prob. squash the commits, so want to get other reviews in first (since I think the commits make it easier to review).

Reply:
I'm indifferent on the name. I think this is fine.